message_ix/tests/test_macro.py

```diff
@@ -272,18 +272,18 @@ def test_calibrate_roundtrip(westeros_solved):
     aeei = with_macro.par("aeei")["value"].values
     npt.assert_allclose(
         aeei,
-        1e-3 * np.array([20, -7.5142661, 43.6256127, 21.14346]),
+        1e-3 * np.array([20, -7.53474955303437, 43.6388021078658, 21.1470758210961]),
```

Comment:
If these numbers are sensitive to the second decimals place, maybe it is better to use a [looser tolerance](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html)

Reply:
Or maybe not - these changes are a result of the factor 1e3 update, right? So perhaps it is necessary to test at this precision if large changes to certain variables affect these values by such a small amount..

Reply:
I would also say that we should keep this for the moment. The change of the aeei values is a result of change above (factor 1e3 update) as you indicated.